adafruit_logging.py

```diff
@@ -187,8 +208,9 @@ def getEffectiveLevel(self) -> int:
         """
         return self._level
 
-    def addHandler(self, handler: LoggingHandler):
+    def addHandler(self, handler: Handler):
```

Comment:
Do  we want to match the argument name `hdlr` from CPython here? 
https://docs.python.org/3/library/logging.html#logging.Logger.addHandler

Personally I think the full word `handler` is nicer, but if we want to match exactly we could take the same name as CPython.

Reply:
Oh good catch!  I can change it to `hdlr`.

Reply:
Okay. I had another look over this today and the rest of it is looking good to me after this arg name, and then the subscript dict access if you want to change that.